source/projects/private-voice-assistant/voice_remote_local_assistant.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: "Configuring a local voice assistant"
```

Comment:
These instructions should be moved into it's own PR. They are also not related to the `projects` but should be generic Assist documentation page (or maybe live on the assist pipeline docs?)

Reply:
```suggestion
title: "Configuring a local assist pipeline"
```